go.mod

```diff
@@ -30,6 +30,30 @@ require (
 	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e // indirect
 	golang.org/x/text v0.3.7 // indirect
 	golang.org/x/time v0.0.0-20220224211638-0e9765cccd65 // indirect
-	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/guregu/null.v3 v3.3.0 // indirect
 )
+
+//
```

Comment:
unless there's a specific reason for it, it's bad form to leave commented code around.  trust your version control.